src/components/layout/tabs/tabs.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Tabs = ({ tabList, activeTab, onSelect }) =>
+  <div className="tabs is-centered is-toggle is-small">
+    <ul className="is-marginless">
+      {tabList.map(tab =>
+        <li
+          className={activeTab === tab.value && 'is-active'}
+          onClick={() => onSelect(tab)}
+          key={tab.value}
+        >
+          <a href={`#${tab.value}`}>
```

Comment:
only thing i wanted to discuss is this href, it's ok to add the #daily or #cumulative hash when the button is pressed? i chose an anchor to match the css framework styles and i receive a React warning if i don't add it.

Reply:
@surdu - can you weigh in on this?

Reply:
i decided to remove the `href` since an `<a>` element without an [href] attribute is still valid. (https://www.w3.org/TR/2011/WD-html5-20110525/text-level-semantics.html#the-a-element)